docs/ml/vision/_index.md

```diff
@@ -18,23 +22,83 @@ While the camera component lets you access what your machine's camera sees, the
 
 Currently, the vision service supports the following kinds of operations:
 
-{{< cards >}}
-{{% card link="/ml/vision/detection/" %}}
-{{% card link="/ml/vision/classification/" %}}
-{{% card link="/ml/vision/segmentation" %}}
-{{< /cards >}}
+- [Detections](#detections)
+- [Classificaitons](#classifications)
+- [Segmentations](#segmentations)
+
+## Detections
+
+<div class="td-max-width-on-larger-screens">
+  <div class="alignright" >
+    {{< imgproc alt="A white dog with a bounding box around it labeled 'Dog: 0.71'" src="/ml/vision/dog-detector.png" resize="300x" declaredimensions=true >}}
+  </div>
+</div>
+
+_2D Object Detection_ is the process of taking a 2D image from a camera and identifying and drawing a box around the distinct "objects" of interest in the scene.
+Any camera that can return 2D images can use 2D object detection.
+
+The service provides different types of detectors, both based on heuristics and machine learning, so that you can create, register, and use detectors for any object you may need to identify.
```

Comment:
(opt) It sounds a little off to me to say "The service provides" when it is different versions of the service. Maybe rephrase like "The service is provided as"?

Reply:
Hmm I don't like the passive voice. How do you feel about "you can use the service as " or " you can use the service with"?